sdk/apimanagement/apimanagement-custom-widget-scaffolder/src/scaffolding.ts

```diff
@@ -39,16 +40,22 @@ export interface TMiscConfig {
 
 export type TConfigs = TWidgetConfig | TDeployConfig | TMiscConfig
 
+/**
+ * @internal
```

Comment:
This is used when the function is exported from a module but it is not exposed publicly in the index.ts. @internal will hide it from the api-extractor output.

Please go through all the interfaces in the API Extractor md file and see if there are any interfaces that can be hiden similarly to this function